easybuild/easyconfigs/p/PETSc/PETSc-3.22.0-foss-2024a.eb

```diff
@@ -0,0 +1,45 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+# Author:    Jasper Grimm (UoY)
+##
+name = 'PETSc'
+version = '3.22.0'
```

Comment:
I realize this is an old PR, but I don't think we should be using a PETSc version that's older than what we're using in the `2023b` generation, see:
- #22696

@maxim-masterov @casparvl Can we bump this version to latest (3.23.5)?

Reply:
We should probably proceed with this PR instead:
- #23252